deps/build.jl

```diff
@@ -47,7 +47,7 @@ if gitver == "notfound"
         isfile(filename) || run(download_cmd(downloadurl, filename))
         # TODO: checksum validation
         (b, ext, sec_ext) = splittarpath(filename)
-        run(unpack_cmd(filename, dest, ext, sec_ext))
+        run(unpack_cmd(filename, dest, is_windows() ? ".7z" : ext, sec_ext))
```

Comment:
Windows can natively work with .7z files? I thought it required 7-Zip or similar. (I could be thinking of a different file type.)